Makefile.common

```diff
@@ -177,6 +177,7 @@ OBJ += frontend/frontend_driver.o \
        $(LIBRETRO_COMM_DIR)/streams/interface_stream.o \
        $(LIBRETRO_COMM_DIR)/streams/memory_stream.o \
        $(LIBRETRO_COMM_DIR)/vfs/vfs_implementation.o \
+       $(LIBRETRO_COMM_DIR)/vfs/vfs_implementation_cdrom.o \
```

Comment:
`vfs_implementation_cdrom` still provides support for bin/cue files when `HAVE_CDROM` is false, so is required whenever `vfs_implementation` is required.